apps/web/src/services/asset-inventory/store/collector-page-store.ts

```diff
@@ -50,5 +52,16 @@ export const useCollectorPageStore = defineStore('collector-page', {
         async setFilteredCollectorList(filters) {
             this.searchFilters = filters;
         },
+        async getCollectorSchedule(id) {
+            try {
+                const res = await SpaceConnector.client.inventory.collector.listSchedules({
+                    collector_id: id,
+                });
+                this.schedules.push(...res.results);
```

Comment:
question: Is there any risk of **duplicate data** being pushed?

Reply:
it will be changed after checking API spec!! So it is not being used within the code.